src/main/java/com/thoughtworks/device/SimulatorManager.java

```diff
@@ -219,16 +247,29 @@ public void deleteSimulator(String deviceName, String osVersion, String osType)
     /**
      * Gets all available Runtimes
      */
-    private List<Runtime> getAllRuntimes() throws IOException, InterruptedException {
+    private List<IOSRuntime> getAllRuntimes() throws IOException, InterruptedException {
         CommandPromptUtil commandPromptUtil = new CommandPromptUtil();
-        String runtimesJSONString = commandPromptUtil.runCommandThruProcess("xcrun simctl list -j runtimes");
+        String runtimesJSONString = commandPromptUtil.runCommandThruProcess("xcrun simctl list -j IOSRuntimes");
```

Comment:
should be `runtimes` here

Reply:
good catch bro .. auto-refactoring are scaring :P